### PR TITLE
Added Achievements and project on blog page

### PR DESCRIPTION
**Pull request by farhanahmedtaj**

### CODE REVIEW

**Comment by s-bonnor** [comment]
Just a reminder that this branch still has merge conflicts

**Comment by uzairzafarr** [comment]
Hello Farhan,

Just to remind you, that this PR still have merge conflict. Kindly fix before Thursday 19 Sep, else this PR will be closed. 

Thanks

File: home/models.py [hunk 1]
```diff
@@ -216,20 +216,27 @@ def __str__(self):
 
         return f'{self.student} - {self.skill.name}: {self.progress}%'
 
-        return f'{self.student} - {self.skill}: {"Completed" if self.completed else "Not completed"}'
```
[the remark this answers is not in the document]
Comment (farhanahmedtaj) [reply home/models.py:219]: Apologies, by mistake I might have removed. I will add it back.

**Review by medwut — APPROVED** [review]
Nice works now fix merge conflict for it to be accepted also for future please add breif description of features and what they do makes it easier for testers to find any issues.

**Comment by farhanahmedtaj** [comment]
Resolved all conflicts.

**Pull request closed without merging**